ztools/xssh/flag.go

```diff
@@ -90,6 +91,42 @@ func (kaList *KexAlgorithmsList) GetStringSlice() []string {
 	}
 }
 
+type CipherList struct {
```

Comment:
IOT use the flag package's Setter pattern, it has to implement the Value interface which a ```[]byte``` doesn't do.